tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/requirements.txt

```diff
@@ -4,7 +4,7 @@ mypy
 pytest
 setuptools>=41.4.0
 wheel
-onnx==1.13.0
+onnx==1.14.0
```

Comment:
It was like:
```
git+http://github.com/onnx/onnx.git@5a5f8a5935762397aa68429b5493084ff970f774#egg=onnx
```
And the commit id should be the one in our onnx submodule.

Reply:
@snnn you are correct. The error is expected as we do not have ONNX 1.14 yet.
I create this PR so that we do not miss this change before 1.15 release.